tools/docker/sanity_check.Dockerfile

```diff
@@ -17,12 +17,12 @@ RUN touch /ok.txt
 # -------------------------------
 FROM python:3.6 as public-api-typed
 
-RUN pip install tensorflow-cpu==2.1.0
-RUN pip install typeguard==2.7.1
-RUN pip install git+https://github.com/gabrieldemarmiesse/typed_api.git@0.1.1
+RUN pip install tensorflow-cpu==2.1.0 \
+			typeguard==2.7.1 \
+			git+https://github.com/gabrieldemarmiesse/typed_api.git@0.1.1
```

Comment:
If I change the version of `typed_api`, it's going to download tensorflow again, which is not really a wanted behavior since it's so big.

Reply:
Yes, in this case(if `typed_api` changed), do you think it's better to clean cache for the `RUN pip install  typedapi==0.2.0` with `--no-cache-dir`, otherwise, the image won't be updated.

Reply:
The image will be updated. Try and see for yourself :)

Reply:
Oh, yes, if you assigned the version number, it'll be updated. But, for `pathlib` and `pyyaml`, in `doc_requirement.txt`,  do we also assign version number?